RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -716,7 +720,7 @@ namespace mkfit {
 
       mhp.reset();
 
-#pragma omp simd
+      //#pragma omp simd doesn't vectorize with current compilers
```

Comment:
is the indentation coming from code-check/code-format?
Having no indent as in the original pragma seems better.